virtualfilesystem.c

```diff
@@ -227,7 +227,7 @@ int is_excluded_from_virtualfilesystem(const char *pathname, int pathlen, int dt
 		return ret;
 	}
 
-	if (dtype == DT_DIR || dtype == DT_LNK) {
```

Comment:
Can a link be a directory or a file?  If the link can be a directory do we need to do something different?

Reply:
The code in treat_one_path() (the normal git excludes logic replaced by the virtual file system hashmap based code) treats DT_REG and DT_LNK the same.  In addition, the comment in the code states "Nobody actually cares about the difference between DT_LNK and DT_REG" so I think we're ok.